### ISSUE TITLE
Episodes removed from In Progress Episodes Reappearing

### ISSUE THREAD

**Issue by inky-pits** [Tikipeter/tikipeter.github.io#75]
I noticed recently in Fen Light when removing episodes to clean up this section, they reappear the next time the menu is loaded. It seems to also happen after clearing all cache.

https://paste.kodi.tv/agozezezey

**Comment by Tikipeter** [comment]
Nope. I don't know what you mean by "removing episodes to clean up this section".

Can you explain it in a different way?

**Comment by inky-pits** [comment]
In the TV Shows section, at the bottom of the screen underneath 'Recently Watched Episodes' there is the 'In Progress Episodes' menu item. After clicking on this and manually removing episode progress for some of the episodes they would normally disappear from 'In Progress Episodes'.

This was useful for example in removing progress of shows I did not intend to continue with. I have noticed that episodes that had their progress manually removed are reappearing. There are also some completed episodes appearing, such as The 8 Show, which I finished and had removed from watch history.

So far I've tried Clear all Fen cache, deleting trakt dB, stopping Kodi app on the Nvidia Shield, clearing Kodi app cache and rebooting. I hadn't thought of rolling back the update. Fen Light is currently on the latest v1.0.64

![PXL_20240806_115037850](https://github.com/user-attachments/assets/e1c54b9d-62ae-49a0-91a1-90e14d6f894a)
![PXL_20240806_115056258](https://github.com/user-attachments/assets/81282eb5-d102-4cff-a88d-1276f9f4b845)
